### PR TITLE
updates to FE, and personalization

### PR DESCRIPTION
**Pull request by JustinParkCS**

### CODE REVIEW

File: cuisino/templates/restaurant/menutest.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{% extends 'users/base.html' %}
```
Comment (JustinParkCS) [inline cuisino/templates/restaurant/menutest.html:1]: This file was used for me to test some stuff. There are no connections to this file.

**Review by FerianiAiden — APPROVED** [review]

**Comment by jmuratov98** [comment]
2 things:
1. lets say i got to /menu/1/ to check out the first item it gets an error
2. In /menu/ can u make each item clickable and goes to the item.

**Comment by JustinParkCS** [comment]
> 2 things:
> 
> 1. lets say i got to /menu/1/ to check out the first item it gets an error

checkout first item? i havent implemented a checkout nor a clicking of the item yet.

> 2. In /menu/ can u make each item clickable and goes to the item.

that can be implemented after? i wanted others to have a easier time adding to the forms and html

**Comment by jmuratov98** [comment]
> checkout first item? i havent implemented a checkout nor a clicking of the item yet.
I don't mean checkout system
image

**Comment by JustinParkCS** [comment]
> > checkout first item? i havent implemented a checkout nor a clicking of the item yet.
> > I don't mean checkout system
> > image

forgot to change that function

**Review by jmuratov98 — APPROVED** [review]

**Pull request merged**